src/WorkItemMigrator/tests/Migration.Common.Tests/RevisionUtilityTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using NUnit.Framework;
+
+using AutoFixture.AutoNSubstitute;
+using AutoFixture;
+using System.Collections.Generic;
+using System;
+using Migration.WIContract;
+
+namespace Migration.Common.Tests
+{
+    [TestFixture]
+    public class RevisionUtilityTests
+    {
+        // use auto fixiture to help mock and instantiate with dummy data with nsubsitute. 
+        private Fixture _fixture;
+
+        [SetUp]
+        public void Setup()
+        {
+            _fixture = new Fixture();
+            _fixture.Customize(new AutoNSubstituteCustomization() { });
+        }
+
+        [Test]
+        public void When_calling_nextvaliddeltarev_with_one_param_Then_the_expected_result_is_returned()
+        {
+            DateTime datetime = new DateTime();
+
+            DateTime expected = datetime + TimeSpan.FromMilliseconds(50);
+            DateTime actual = RevisionUtility.NextValidDeltaRev(datetime);
+
+            Assert.AreEqual(expected, actual);
+        }
+
+        [Test]
+        public void When_calling_nextvaliddeltarev_with_next_more_than_current_Then_the_expected_result_is_returned()
+        {
+            DateTime datetime1 = new DateTime();
+            DateTime datetime2 = datetime1 + TimeSpan.FromMilliseconds(60);
+
+            DateTime expected = datetime1 + TimeSpan.FromMilliseconds(50);
+            DateTime actual = RevisionUtility.NextValidDeltaRev(datetime1, datetime2);
+
+            Assert.AreEqual(expected, actual);
+        }
+
+        [Test]
+        public void When_calling_replacehtmlelements_Then_the_expected_result_is_returned()
+        {
+            string expected = "html";
+            string actual = RevisionUtility.ReplaceHtmlElements("html");
```

Comment:
Test two strings that comform to the tested patterns instead, not just "html"
See:
  string imageWrapPattern = "<span class=\"image-wrap\".*?>.*?(<img .*? />).*?</span>";
  string userLinkPattern = "<a href=.*? class=\"user-hover\" .*?>(.*?)</a>";

Reply:
What would be a good string value to test? I'm having a hard time understandning what Regex.Replace does.

Reply:
You can use https://regex101.com/ to verify the regex expression against a test string.

Here's two string that would work, or smth similar. Test both cases :)

- `<a href=https://text.com class="user-hover" >placeholder string</a>`
- `<span class=\"image-wrap\">(<img src="img.jpg" />)</span>`

Reply:
Added two tests, please take a look when you have time.